impexp/README.md

```diff
@@ -0,0 +1,33 @@
+# Generating ucumDefs.json
```

Comment:
I think this file needs to be mentioned in ../README.md.  There, how about:

+++Building the code and data
If you wish to modify the code, the build process is simply:
```
  npm run build
```
If you wish to modify or update either of the source data files (ucum-essence.xml or ucum.csv), see impexp/README.md for instructions on build steps for data changes.

Reply:
Done